.github/workflows/test-k8s-service-buffer.yml

```diff
@@ -242,16 +239,13 @@ jobs:
         if: always()
         run: |
           set -Eeuo pipefail
-          
```

Comment:
Why you are removing these empty lines?

Reply:
Jachym was last time complaining about it in PR. I have set up my editor according to `editorconfig` and done this. I can drop the commit if we want, just want to point out that according to editorconfig the trailing spaces are removed.
As I found on this line newline and trailing spaces I just removed the newline.
As I pointed out above, i can drop the commit :slightly_smiling_face:

Reply:
:shrug: I understand if it removed trailing spaces but why should it delete empty lines in the middle of the code?

Reply:
I understand, maybe my bad, I'll make sure next time leave new lines :slightly_smiling_face: .